packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.ts

```diff
@@ -172,6 +167,36 @@ export function trackClickActions(
   }
 }
 
+function computeClickActionBase(event: MouseEvent & { target: Element }, actionNameAttribute?: string) {
+  let target: ClickAction['target']
+  let position: ClickAction['position']
+
+  if (isExperimentalFeatureEnabled('clickmap')) {
+    const rect = event.target.getBoundingClientRect()
+    target = assign(
+      {
+        width: Math.round(rect.width),
+        height: Math.round(rect.height),
+      },
+      getSelectorsFromElement(event.target, actionNameAttribute)
+    )
+    position = {
+      // Use clientX and Y because for SVG element offsetX and Y are relatives to the <svg> element
+      x: Math.round(event.clientX - rect.left),
+      y: Math.round(event.clientY - rect.top),
+    }
+  }
+
+  return {
+    type: 'click',
+    target,
+    position,
+    name: getActionNameFromElement(event.target, actionNameAttribute),
+    event,
+    startClocks: clocksNow(),
+  }
+}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->👏